### DISCUSSION TITLE
Error: Invalid task point declaration at line 18: C2327856000S00016433ESTART��

### DISCUSSION THREAD

**Discussion by JuanIrache** [Turbo87/igc-parser#672] [Q&A] [answered]
Hello,

Thank you for the great module. I am facing a problem with one log file. The module crashes when it encounters this line

> C2327856000S00016433ESTART��

> Error: Invalid task point declaration at line 18: C2327856000S00016433ESTART��

Here's the rest of the header (line 19 after it also causes the crash)

> AXFI000914
> HFDTEDATE:250322
> HFPLTPILOTINCHARGE:NIL
> HFCM2CREW2:NIL
> HFGTYGLIDERTYPE:NKN
> HFGIDGLIDERID:NKN
> HFDTMGPSDATUM:WGS84
> HFRFWFIRMWAREVERSION:2.03g
> HFRHWHARDWAREVERSION:1.0
> HFFTYFRTYPE:Flymaster,GPSSD,B203
> HFGPSRECEIVER:UBLOX,NEO8,56,50000,GPS,GLO,GAL
> HFPRSPRESSALTSENSOR:FREESCALE,MPXA6115A,13608
> HFALGALTGPS:GEO
> HFALPALTPRESSURE:ISA
> LPLTFlymasterDesigner 2.04e
> C250322114838000000000000
> C0000000N00000000WTAKEOFF
> C2327856000S00016433ESTART��
> C2327856000S00016433EFINISH��
> C0000000N00000000WLANDING
> B1148384947168N00118170EA0004800154

Can you tell if this is a problem with the file format or it can potentially be parsed?

I can provide the entire file if necessary.

Thank you.

**Comment by Turbo87** [comment, marked as answer]
![Bildschirmfoto 2025-01-06 um 16 05 09](https://github.com/user-attachments/assets/9cb8de41-f631-4375-b773-82b82f025e07)

These are the valid characters in an IGC file according to the specification. It looks like your file is using invalid characters.
